spring-cloud-dataflow-common/spring-cloud-dataflow-common-test-docker/src/main/java/org/springframework/cloud/dataflow/common/test/docker/compose/execution/Command.java

```diff
@@ -50,8 +50,8 @@ public Command(Executable executable, Consumer<String> logConsumer) {
 		this.logConsumer = logConsumer;
 	}
 
-	public String execute(ErrorHandler errorHandler, String... commands) throws IOException, InterruptedException {
-		ProcessResult result = run(commands);
+	public String execute(ErrorHandler errorHandler, boolean composeCommand, String... commands) throws IOException, InterruptedException {
```

Comment:
DockerTests mocks need to be updated with the method signature.

Reply:
Done